packages/amplify_datastore/android/src/main/kotlin/com/amazonaws/amplify/amplify_datastore/FlutterModelProvider.kt

```diff
@@ -19,14 +19,15 @@ import com.amplifyframework.core.model.Model
 import com.amplifyframework.core.model.ModelProvider
 import com.amplifyframework.core.model.ModelSchema
 import com.amplifyframework.util.Immutable
+import java.util.Collections
 
 class FlutterModelProvider private constructor() : ModelProvider {
     private val modelSchemaMap: MutableMap<String, ModelSchema> =
             HashMap()
 
     // TODO change it to return an empty set once it's properly supported in the amplify-android library
     override fun models(): Set<Class<out Model?>>? {
-        return null
+        return Collections.emptySet()
```

Comment:
We already [made this change](https://github.com/aws-amplify/amplify-flutter/blob/fb16d0597c36559a20a1d9801bf0fe3c2106ebc8/packages/amplify_datastore/android/src/main/kotlin/com/amazonaws/amplify/amplify_datastore/FlutterModelProvider.kt#L29).

Maybe you need to rebase this branch?

Please remove the TODO comment, though.

Reply:
removed the comments.  rebase has happened

Reply:
I still see this change.